### PR TITLE
ColorBox: add Vue and React code snippets

### PR DESCRIPTION
**Pull request by ksercs**

### CODE REVIEW

File: concepts/05 Widgets/ColorBox/10 Handle the Value Change Event.md [hunk 1]
```diff
@@ -76,6 +120,63 @@ To process a new **ColorBox** value, you need to handle the value change event.
         // ...
     })
 
+##### Vue
+
+    <!-- tab: App.vue -->
+    <template>
+        <DxColorBox
+            :onValueChanged="colorBox_valueChanged"
```
Comment (AlexJameson) [inline concepts/05 Widgets/ColorBox/10 Handle the Value Change Event.md]:
```suggestion
            @value-changed="handleValueChange"
```

File: concepts/05 Widgets/ColorBox/10 Handle the Value Change Event.md [hunk 2]
```diff
@@ -76,6 +120,63 @@ To process a new **ColorBox** value, you need to handle the value change event.
...
+
+    import DxColorBox from 'devextreme-vue/color-box';
+
+    export default {
+        components: {
+            DxColorBox
+        },
+        data() {
+            return {
+                colorBox_valueChanged: function (e) {
```
Comment (AlexJameson) [inline concepts/05 Widgets/ColorBox/10 Handle the Value Change Event.md]:
```suggestion
                handleValueChange: function (e) {
```

File: concepts/05 Widgets/ColorBox/10 Handle the Value Change Event.md [hunk 3]
```diff
@@ -76,6 +120,63 @@ To process a new **ColorBox** value, you need to handle the value change event.
...
+
+    <!-- tab: App.js -->
+    import React from 'react';
+    import 'devextreme/dist/css/dx.common.css';
+    import 'devextreme/dist/css/dx.light.css';
+
+    import ColorBox from 'devextreme-react/color-box';
+
+    class App extends React.Component {
+        colorBox_valueChanged(e) {
```
Comment (AlexJameson) [inline concepts/05 Widgets/ColorBox/10 Handle the Value Change Event.md]: I suggest renaming the function in React and Angular approaches based on my suggestion above.

File: concepts/05 Widgets/ColorBox/15 Support Alpha Channel.md [hunk 4]
```diff
@@ -33,6 +33,50 @@ By default, the **ColorBox** does _not_ allow an end user to control the transpa
         // ...
     })
 
+##### Vue
+
+    <!-- tab: App.vue -->
+    <template>
+        <DxColorBox
+            value="rgba(255, 144, 0, 0.3)"
```
Comment (AlexJameson) [inline concepts/05 Widgets/ColorBox/15 Support Alpha Channel.md]: Please use two-way binding as in the Angular approach.

File: concepts/05 Widgets/ColorBox/15 Support Alpha Channel.md [hunk 5]
```diff
@@ -33,6 +33,50 @@ By default, the **ColorBox** does _not_ allow an end user to control the transpa
...
+    import 'devextreme/dist/css/dx.common.css';
+    import 'devextreme/dist/css/dx.light.css';
+
+    import ColorBox from 'devextreme-react/color-box';
+
+    class App extends React.Component {
+        render() {
+            return (
+                <ColorBox
+                    value="rgba(255, 144, 0, 0.3)"
```
Comment (AlexJameson) [inline concepts/05 Widgets/ColorBox/15 Support Alpha Channel.md]: Set this value using the onValueChanged handler to imitate the binding. Refer to [this demo](https://js.devexpress.com/Demos/WidgetsGallery/Demo/ColorBox/Overview/React/Light/) for an example.

**Review by AlexJameson — CHANGES_REQUESTED** [review]

File: concepts/05 Widgets/ColorBox/10 Handle the Value Change Event.md (hunk 1 above)
Comment (ksercs) [reply concepts/05 Widgets/ColorBox/10 Handle the Value Change Event.md]: fixed

File: concepts/05 Widgets/ColorBox/10 Handle the Value Change Event.md (hunk 2 above)
Comment (ksercs) [reply concepts/05 Widgets/ColorBox/10 Handle the Value Change Event.md]: fixed

File: concepts/05 Widgets/ColorBox/10 Handle the Value Change Event.md (hunk 3 above)
Comment (ksercs) [reply concepts/05 Widgets/ColorBox/10 Handle the Value Change Event.md]: fixed

File: concepts/05 Widgets/ColorBox/15 Support Alpha Channel.md (hunk 4 above)
Comment (ksercs) [reply concepts/05 Widgets/ColorBox/15 Support Alpha Channel.md]: fixed

File: concepts/05 Widgets/ColorBox/15 Support Alpha Channel.md (hunk 5 above)
Comment (ksercs) [reply concepts/05 Widgets/ColorBox/15 Support Alpha Channel.md]: fixed

File: concepts/05 Widgets/ColorBox/10 Handle the Value Change Event.md [hunk 6]
```diff
@@ -55,14 +99,14 @@ To process a new **ColorBox** value, you need to handle the value change event.
 
     <!--HTML-->
     <dx-color-box ...
-        (onValueChanged)="colorBox_valueChanged($event)">
+        (onValueChanged)="handleValueChanged($event)">
```
Comment (AlexJameson) [inline concepts/05 Widgets/ColorBox/10 Handle the Value Change Event.md]: I believe you have a typo here, see the screenshot. Please rename here and below
image

File: concepts/05 Widgets/ColorBox/15 Support Alpha Channel.md [hunk 7]
```diff
@@ -33,6 +33,72 @@ By default, the **ColorBox** does _not_ allow an end user to control the transpa
         // ...
     })
 
+##### Vue
+
+    <!-- tab: App.vue -->
+    <template>
+        <DxColorBox
+            :value.sync="value"
```
Comment (AlexJameson) [inline concepts/05 Widgets/ColorBox/15 Support Alpha Channel.md]: Let's call this variable 'color' to make it more meaningful. Apply here and below in Vue and React.

**Review by AlexJameson — CHANGES_REQUESTED** [review]

File: concepts/05 Widgets/ColorBox/10 Handle the Value Change Event.md (hunk 6 above)
Comment (ksercs) [reply concepts/05 Widgets/ColorBox/10 Handle the Value Change Event.md]: fixed

File: concepts/05 Widgets/ColorBox/15 Support Alpha Channel.md (hunk 7 above)
Comment (ksercs) [reply concepts/05 Widgets/ColorBox/15 Support Alpha Channel.md]: fixed

**Review by AlexJameson — APPROVED** [review]

File: concepts/05 Widgets/ColorBox/10 Handle the Value Change Event.md [hunk 8]
```diff
@@ -44,8 +109,8 @@ To process a new **ColorBox** value, you need to handle the value change event.
     $(function() {
         $("#colorBoxContainer").dxColorBox({
             onValueChanged: function (e) {
-                var previousValue = e.previousValue;
-                var newValue = e.value;
+                let previousValue = e.previousValue;
```
Comment (RomanTsukanov) [inline concepts/05 Widgets/ColorBox/10 Handle the Value Change Event.md]: Let's change the `let`s to `const`s.

**Review by RomanTsukanov — CHANGES_REQUESTED** [review]

File: concepts/05 Widgets/ColorBox/10 Handle the Value Change Event.md (hunk 8 above)
Comment (ksercs) [reply concepts/05 Widgets/ColorBox/10 Handle the Value Change Event.md]: fixed

File: concepts/05 Widgets/ColorBox/10 Handle the Value Change Event.md [hunk 9]
```diff
@@ -4,12 +4,12 @@ By default, the **ColorBox** applies value after an end user clicks the "Apply"
...
-        $("#colorBoxContainer").dxColorBox({
-            value: "#FF0000",
-            applyValueMode: 'instantly'
+        $(function() {
+            $("#colorBoxContainer").dxColorBox({
+                value: "#FF0000",
+                applyValueMode: 'instantly'
+            });
         });
-    });
```
Comment (RomanTsukanov) [inline concepts/05 Widgets/ColorBox/10 Handle the Value Change Event.md]: This example is misaligned now. Let's revert these changes.

File: concepts/05 Widgets/ColorBox/10 Handle the Value Change Event.md [hunk 10]
```diff
@@ -76,6 +141,63 @@ To process a new **ColorBox** value, you need to handle the value change event.
         // ...
     })
 
+##### Vue
+
+    <!-- tab: App.vue -->
+    <template>
+        <DxColorBox
+            @value-changed="handleValueChange"
```
Comment (RomanTsukanov) [inline concepts/05 Widgets/ColorBox/10 Handle the Value Change Event.md:153]: Same notes here as in the [DateBox PR](https://github.com/DevExpress/devextreme-documentation/pull/362#pullrequestreview-362500110).

**Review by RomanTsukanov — CHANGES_REQUESTED** [review]

File: concepts/05 Widgets/ColorBox/10 Handle the Value Change Event.md (hunk 9 above)
Comment (RomanTsukanov) [reply concepts/05 Widgets/ColorBox/10 Handle the Value Change Event.md]: The changes are still present.

**Review by RomanTsukanov — CHANGES_REQUESTED** [review]

File: concepts/05 Widgets/ColorBox/10 Handle the Value Change Event.md (hunk 9 above)
Comment (ksercs) [reply concepts/05 Widgets/ColorBox/10 Handle the Value Change Event.md]: fixed

**Review by RomanTsukanov — APPROVED** [review]
